src/coreComponents/physicsSolvers/wavePropagation/WaveSolverUtils.hpp

```diff
@@ -289,8 +292,67 @@ struct WaveSolverUtils
 
 
 
-};
+/**
+ * Update pressure in one direction
+ */
+  static void UpdateP( NodeManager & nodeManager,
```

Comment:
Seems ok to me! 
Just, I don't know if I would have called it "UpdateP" because when we go to elastic it is not the pressure which is updated. Maybe a name in link with dt like "TimeUpdate" or somethin like that I don't know, what do you think @sframba ? 
It is not a big deal anyway we can change that later (I can do it in my refactoring after if needed)

Reply:
Indeed we can rename it in the PR or later.